src/prelude.js

```diff
@@ -59,6 +60,17 @@ internals.forcemv = function(x) {
   return typeof x == 'object' && x !== null && 'multiple-value' in x? x: internals.mv(x);
 };
 
+internals.error = function(){
+  var args = Array.prototype.slice.call(arguments);
```

Comment:
Should we start using `...args` in arguments already?

Reply:
Yes, I was just about to ask this question :D
I can do another revision that use `...args` for everything.

Reply:
Also should we consider compiling regular user CL functions to use spread syntax instead of `arguments` object as well? Although this should probably be done in the *lambda list overhaul* project.

Reply:
Let's try to do a few iterations where we modernize the JS we both write and generate I'd say, yes.